parser/index.ts

```diff
@@ -1,6 +1,5 @@
-// @ts-expect-error - to review how it's implemented in ExpensiMark
 // eslint-disable-next-line import/no-unresolved
-import {ExpensiMark} from 'expensify-common/lib/ExpensiMark';
+import {ExpensiMark} from 'expensify-common';
```

Comment:
```suggestion
import ExpensiMark from 'expensify-common/lib/ExpensiMark';
```

Reply:
This import now isn't correct when install `Expensify-common` by version.

Reply:
Fixed in 1cfd1eb.

Reply:
Actually I had to revert this change as it caused the whole `expensify-common` to bundle and thus significantly increase its bundle size. I guess it's okay to leave it as it is now since the types also work correctly.